### PR TITLE
[RHODS-12555] - CVE-2023-44487

### PR DESCRIPTION
**Pull request by spolti**
## Description

Update affected dependencies of the https://github.com/advisories/GHSA-qppj-fm5r-hxr3

## How Has This Been Tested?

Snyk scan result:
```
$ snyk test
...
✔ Tested 707 dependencies for known issues, no vulnerable paths found.
```

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: config/rbac/role.yaml [hunk 1]
```diff
@@ -111,7 +111,6 @@ rules:
   - networking.k8s.io
   resources:
   - networkpolicies
-  - ingresses
```
Comment (spolti) [inline config/rbac/role.yaml:114]: @vaibhavjainwiz the changes in this file was done automatically just by running `make test`.

Is this expected?

Comment (VedantMahabaleshwarkar) [reply config/rbac/role.yaml:114]: This PR should not change manifests I think

**Review by VedantMahabaleshwarkar — CHANGES_REQUESTED** [review]
role.yaml is changed.. it shouldn't be

**Comment by spolti** [comment]
I guess the CI is failing due the same issue that @vaibhavjainwiz mentioned earlier this week:

```
TAG=fast is set
.. Updating repo uri in /tmp/modelmesh-e2e/kfdef.yaml
REPO-URI=local is set
```

**Comment by Jooho** [comment]
did fvt test on my cluster and it worked so merge manually.

**Review by Jooho — APPROVED** [review]
/lgtm

**Pull request merged**